src/scenarios/emptyconsoletemplateinnerloop/test.py

```diff
@@ -0,0 +1,14 @@
+'''
+C# Console app
+'''
+from shared.runner import TestTraits, Runner
+
+EXENAME = 'emptyconsoletemplateinnerloop'
+
+if __name__ == "__main__":
+    traits = TestTraits(exename=EXENAME, 
+                        startupmetric='InnerLoop',
+                        guiapp='false',
+                        innerloopcommandargs='-c "from shutil import copyfile; copyfile(\'src/Program.cs\', \'app/Program.cs\')"' 
```

Comment:
should this use pathsep?

Reply:
Yeah, I should probably use os.join to create these paths.